test/extended/builds/digest.go

```diff
@@ -70,6 +70,7 @@ func testBuildDigest(oc *exutil.CLI, buildFixture string, buildLogLevel uint) {
 		logLevelArg := fmt.Sprintf("--build-loglevel=%d", buildLogLevel)
 		g.By("starting a test build")
 		br, err := exutil.StartBuildAndWait(oc, "test", logLevelArg)
```

Comment:
Why is this err ignored?

Reply:
good question. it shouldn't have been. updated.